WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/viewholders/TextViewHolder.kt

```diff
@@ -8,19 +8,18 @@ import android.text.SpannableString
 import android.text.TextPaint
 import android.text.method.LinkMovementMethod
 import android.text.style.ClickableSpan
+import android.text.style.StyleSpan
 import android.view.View
 import android.view.ViewGroup
 import android.widget.TextView
-import org.wordpress.android.R.color
-import org.wordpress.android.R.id
-import org.wordpress.android.R.layout
+import org.wordpress.android.R
 import org.wordpress.android.ui.stats.refresh.lists.sections.BlockListItem.Text
 
 class TextViewHolder(parent: ViewGroup) : BlockListItemViewHolder(
         parent,
-        layout.stats_block_text_item
+        R.layout.stats_block_text_item
```

Comment:
`R.` was added here, but it's missing in `TagViewHolder`, `ImageItemViewHolder` and `BigTitleViewHolder`